sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -161,9 +168,14 @@ func App() *cli.App {
 					Destination: &useCachedImage,
 					Value:       false,
 				},
+				&cli.StringSliceFlag{
+					Name:  "label",
+					Usage: "add multiple labels to the pulled image. The label should be in the format of `key=value`",
+				},
 			},
 			Action: func(c *cli.Context) error {
-				return pullImageOnly(containerdSocket, namespace, source, registryConfig, useCachedImage)
+				labels := strings.Join(c.StringSlice("label"), `, `)
```

Comment:
Why take a slice of strings and turn it into a single string, only to split it back out later? Can you just pass around the slice and avoid the `strings.Join` and `strings.Split` calls?

Reply:
Yeah, good catch. I change to use the slice.